com.github.nrittsti.NTag.yml

```diff
@@ -0,0 +1,32 @@
+app-id: com.github.nrittsti.NTag
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
```

Comment:
Update to runtime 20.08

Reply:
Thanks for the review. I only looked at the Flatpak documentation and not at the releases page of the runtime.